src/mixins/data-iterable.js

```diff
@@ -45,9 +45,9 @@ export default {
       type: Array,
       default () {
         return [
-          5,
-          10,
-          25,
+          { value: 5, text: 5 },
+          { value: 10, text: 10 },
+          { value: 25, text: 25 },
```

Comment:
Why do these need to be objects?

Reply:
> Why do these need to be objects?

In the `computedRowsPerPageItems`method, if it's an object the `text` will be passed to the `t` method and since the text is a number instead of a string, it get's localized in the 'if' statement I added.

Reply:
Wouldn't it make more sense to pass it through `t()` on the other side of that ternary too?

Reply:
No, beacause then the values could be non-english digit charcters and it won't work in the rest of the code where the actual numbers are needed. Coercion of string to number only happens for strings of ascii digits.
We could however convert it to an object inside the computed method instead of changing the default items to objects